### ISSUE TITLE
validate role fails on Tails 4

### ISSUE THREAD

**Issue by redshiftzero** [freedomofpress/securedrop#4767]
## Description

validate role fails on Tails 4

## Steps to Reproduce

1. Install the Tails 4 beta: https://tails.boum.org/news/test_4.0-beta2/
2. Set up persistence
3. Clone the SecureDrop repo
4. From ~/Persistent/securedrop, run `./securedrop-admin tailsconfig` (after the workaround for `securedrop-admin setup` documented in #4751)

## Expected Behavior

tasks in validate role should pass

## Actual Behavior

```
TASK [tails-config : Confirm host OS is Tails.] ******************************************************************************************************
fatal: [localhost]: FAILED! => {
    "assertion": "ansible_lsb.id == \"Tails\"", 
    "changed": false, 
    "evaluated_to": false, 
    "msg": "SecureDrop requires Tails 3 or greater for workstation environments."
}
```

## Comments

Adding a debug step reveals:

```
TASK [tails-config : print ansible_lsb] ***************************************************************************************************************************
ok: [localhost] => {
    "ansible_lsb": {
        "codename": "buster", 
        "description": "Debian GNU/Linux 10 (buster)", 
        "id": "Debian", 
        "major_release": "10", 
        "release": "10"
    }
}
```

Related to #4669

**Comment by redshiftzero** [comment]
the root cause here also causes #4751 (securedrop-admin does not use torify if `lsb_release` is not Tails)

**Comment by zenmonkeykstop** [comment]
/etc/os-release seems to be a better source here.

**Comment by zenmonkeykstop** [comment]
FYI, I filed a bug for this with the Tails project. `lsb_release` seems to return the wrong values on Tails 4.0. I'll see if there's a workaround we can use in the meantime.

https://redmine.tails.boum.org/code/issues/17074

**Comment by zenmonkeykstop** [comment]
This was resolved by #4852
